### PR TITLE
Implement sc name change and add new sc

### PR DESCRIPTION
**Pull request by Bhagyashreek8**
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.
- [x] I have made sure existing UTs are not impacted
- [x] I have executed all necessary linter and made sure internal pipeline/travis is not broken
https://v3.travis.ibm.com/github/alchemy-containers/ibm-object-csi-driver/jobs/19091672

### CODE REVIEW

**Comment by Bhagyashreek8** [comment]
Test results here - https://github.ibm.com/alchemy-containers/armada-storage/issues/5985#issuecomment-79473398

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: ashimagarg/csi
-  newTag: "53"
+  newName: bhagyak1/ibm-object-csi-driver-operator
+  newTag: m1402
```
Comment (nkkashyap) [inline config/manager/kustomization.yaml:8]: Please avoid personal registry and tags here.

**Review by nkkashyap — COMMENTED** [review]
code change looks good

File: config/manager/kustomization.yaml (hunk 1 above)
Comment (Bhagyashreek8) [reply config/manager/kustomization.yaml]: yes removed it.

File: controllers/ibmobjectcsi_controller_test.go [hunk 2]
```diff
@@ -809,7 +864,6 @@ func TestIBMObjectCSIReconcile(t *testing.T) {
 			assert.Equal(t, testcase.expectedResp, res)
 
 			if testcase.expectedErr != nil {
-				assert.Error(t, err)
```
[the remark this answers is not in the document]
Comment (Bhagyashreek8) [reply controllers/ibmobjectcsi_controller_test.go]: Removed by miss. Added it back. Thanks @ashimagarg27

**Comment by Bhagyashreek8** [comment]
UT coverage 
Screenshot 2024-05-15 at 11 20 14 AM

**Review by ashimagarg27 — APPROVED** [review]

**Review by ambiknai — APPROVED** [review]

**Pull request merged**